sdcm/cluster_k8s/__init__.py

```diff
@@ -2459,6 +2460,30 @@ def seed_nodes_ips(self):
     def seed_nodes(self):
         return []
 
+    @property
+    def connection_bundle_file(self) -> str | None:
```

Comment:
It is always better to reduce code indentation when possible:
```
@property
def connection_bundle_file(self) -> str | None:
    if bundle_file := super().connection_bundle_file:
        return bundle_file

    bundle_cmd_output = self.k8s_cluster.kubectl(
        f"get secret/{self.scylla_cluster_name}-local-cql-connection-configs-admin"
        f" --template='{{{{ index .data \"{self.scylla_cluster_name}.sct.scylladb.com\" }}}}'",
        namespace=self.namespace, ignore_status=True)
    if !bundle_cmd_output.ok:
        return None

    bundle_file = Path(tempfile.mktemp())
    bundle_file.write_bytes(base64.decodebytes(
        bytes(bundle_cmd_output.stdout.strip(), encoding='utf-8')))
    lb_cluster_ip = self.k8s_cluster.kubectl(
        "get service/haproxy-kubernetes-ingress --template='{{ index .spec.clusterIP }}'",
        namespace="haproxy-controller")
    if lb_cluster_ip.ok:
        bundle_yaml = yaml.safe_load(bundle_file.open("r"))
        for dc_name, connection_data in bundle_yaml.get('datacenters', {}).items():
	    connection_data['server'] = lb_cluster_ip.stdout.strip()
        yaml.dump(bundle_yaml, bundle_file.open('w'))
    return bundle_file
```

Reply:
fixed